### PR TITLE
Cleanup workspace warnings

### PR DESCRIPTION
**Pull request by arturkamieniecki**
I've cleaned up some warnings form the compilation of the ros2_ws.

### CODE REVIEW

File: ros2_ws/src/blind_path_follower/src/follow_path.cpp [hunk 1]
```diff
@@ -128,6 +128,7 @@ class FollowPathActionServer
     rclcpp_action::GoalResponse handle_goal(const rclcpp_action::GoalUUID& uuid, std::shared_ptr<const FlwPthAction::Goal> goal)
     {
         (void)uuid;
+        (void)goal;
```
Comment (jhanca-robotecai) [inline ros2_ws/src/blind_path_follower/src/follow_path.cpp:131]: Just out of curiosity: why not `[maybe_unused]`?

**Review by jhanca-robotecai — CHANGES_REQUESTED** [review]
Warnings have a meaning and we should either fix the issues or keep the warnings so we know there are issues. Calling `void(variable_name)` hides the warnings, but does not solve the issues. 

What you did in `palletizationNode.cpp` is valid, but the other two changes are not.

**Comment by adamdbrw** [comment]
Change (void) to maybe unused for unused parameters and we can merge this.

**Comment by jhanca-robotecai** [comment]
> Change (void) to maybe unused for unused parameters and we can merge this.

The reason for this parameter to be unused is that it was set fixed within the method. The parameter should be removed completely if not needed and the function calls should be modified accordingly.

Adding `[[maybe_unused]]` is like:
```cpp
int add([[maybe_unused]]int a, [[maybe_unused]]int b) {
    return 2;
}
```

**Review by michalpelka — APPROVED** [review]

**Pull request merged**